core/Model/CalendarModel.vala

```diff
@@ -60,20 +60,25 @@ public class Maya.Model.CalendarModel : Object {
     private E.CredentialsPrompter credentials_prompter;
 
     private static Maya.Model.CalendarModel? calendar_model = null;
+    private static GLib.Settings state_settings;
 
     public static CalendarModel get_default () {
         if (calendar_model == null)
             calendar_model = new CalendarModel ();
         return calendar_model;
     }
 
+    static construct {
+        state_settings = new GLib.Settings ("io.elementary.calendar.savedstate");
```

Comment:
This is the same as `Application.saved_state`. Do we need a second instance?

Reply:
Yeah this is in the library though so I don’t think it has access to the application

Reply:
Ah, OK. Although presumable the application has access to the library so we may still not need two instances?

Reply:
Yeah that’s valid. Could get things from calendarmodel instead

Reply:
Can be done in another PR